src/aot/Akka.AOT.App/Program.cs

```diff
@@ -0,0 +1,27 @@
+﻿using Akka.Actor;
+using Akka.AOT.App.Actors;
+
+namespace Akka.AOT.App;
+
+class Program
+{
+    static async Task Main(string[] args)
+    {
+        var system = ActorSystem.Create("MySystem");
```

Comment:
Does some end to end messaging and uses the default configuration loading, which currently triggers a number of AOT warnings due to how we load `Type`s from HOCON strings - those are all issues on the https://github.com/akkadotnet/akka.net/issues/7246 epic we will have to gradually fix before this spec can pass.